molgenis-jobs/src/main/java/org/molgenis/jobs/ProgressImpl.java

```diff
@@ -98,7 +98,7 @@ public void failed(Exception ex) {
     JOB_EXECUTION_LOG.error("Failed. " + ex.getMessage(), ex);
     jobExecution.setEndDate(Instant.now());
     jobExecution.setStatus(FAILED);
-    jobExecution.setProgressMessage(ex.getMessage());
+    jobExecution.setProgressMessage(ex.getLocalizedMessage());
```

Comment:
I think this:
- should be in a separate PR with GitHub issue
- needs a unit test